### PR TITLE
IGAPP-1139: Change footer seperation line color to #B1B1B1

### PR DESCRIPTION
**Pull request by randowha**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]
Nicely done! Tested on Firefox

**Review by steffenkleinle — APPROVED** [review]
Congrats to your first PR, thank you! :tada: :rocket: Once you applied all prettier changes you are good to merge :)

**Comment by steffenkleinle** [comment]
@randowha on a minor note: We try to prefix all our commits with the issue number, in your case this would be for example:
`IGAPP-1139: Change footer seperation line color to #B1B1B1`

**Review by steffenkleinle — CHANGES_REQUESTED** [review]
Looks like your merge somehow failed :/ Also, Why did you revert the usage of our textColor constant back to a magic value?

**Comment by randowha** [comment]
> 

would

**Comment by randowha** [comment]
> Looks like your merge somehow failed :/ Also, Why did you revert the usage of our textColor constant back to a magic value?

There is no color #b1b1b1 yet. Would you like me to use the already existing textDecorationColor (#c7c7c7) or to create a new color constant?

File: web/src/components/CityNotCooperatingFooter.tsx.orig [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  background-color: ${props => props.theme.colors.backgroundAccentColor};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+<<<<<<< HEAD
+  border-bottom: 2px solid #b1b1b1;
+=======
+  border-bottom: 2px solid #B1B1B1;
+>>>>>>> a77968e42d7dc6205176b74dc1798a2bf95c2d01
```
Comment (LeandraH) [inline web/src/components/CityNotCooperatingFooter.tsx.orig:20]: Looks like a failed merge here. I think the accepted one should be the lower case one, right?

**Comment by LeandraH** [comment]
> > Looks like your merge somehow failed :/ Also, Why did you revert the usage of our textColor constant back to a magic value?
> 
> There is no color #b1b1b1 yet. Would you like me to use the already existing textDecorationColor (#c7c7c7) or to create a new color constant?

Hmmm, good question, we are now using that color in two places. My usual rule of thumb is to make it a constant after three repeats but we can also make one here already?

**Comment by randowha** [comment]
Sorry for all these commits, but because of merging main into my branch in the middle of the feature, I find myself unable to squash them together. If sb is able to resolve that, feel free

File: build-configs/common/theme/colors.ts [hunk 2]
```diff
@@ -19,6 +20,7 @@ export const commonLightColors = {
   textSecondaryColor: '#585858',
   textDecorationColor: '#c7c7c7',
   textDisabledColor: '#d0d0d0',
+  footerLineColor: 'b1b1b1',
```
Comment (steffenkleinle) [inline build-configs/common/theme/colors.ts:23]: Even so it works fine, I would prefer the hash-prefixed variant for consistency:
```suggestion
  footerLineColor: '#b1b1b1',
```

**Review by steffenkleinle — APPROVED** [review]

**Review by steffenkleinle — APPROVED** [review]

**Pull request merged**